config/_default/menus.toml

```
@@ -16,6 +16,12 @@
   identifier = "plugins"
   url = "/docs/plugins/"
 
+[[main]]
+  name = "Tutorials"
+  weight = 20
+  identifier = "tutorials"
+  url = "/blog/"
```

Cannot we use them as?

```suggestion
  url = "/tutorials/"
```